setup.sh

```diff
@@ -0,0 +1,33 @@
+#/bin/bash
```

Comment:
Please add a comment explaining what this script does and an example invocation.

Reply:
Done.